nbformat/validator.py

```diff
@@ -241,7 +229,7 @@ def better_validation_error(error, version, version_minor):
 
 
 def validate(nbdict=None, ref=None, version=None, version_minor=None,
-             relax_add_props=False, nbjson=None):
+             relax_add_props=False, nbjson=None, use_fast=False):
```

Comment:
maybe default `use_fast` to `None` so you can default to the environment variable version unless it's explicitly turned off or on?

Reply:
Oops this is a leftover from the previous approach. Can be removed.

Reply:
Fixed.